### DISCUSSION TITLE
Uploading a CSV for a collection

### DISCUSSION THREAD

**Discussion by arpijoy** [directus/directus#6194] [Feature Requests]
I would really like to see an upload functionality, like a DB CSV import done via directus

**Comment by rijkvanzanten** [comment]
Like this? https://docs.directus.io/reference/api/system/utilities/#import-data-from-file

**Reply by arpijoy** [reply]
I see that there is a utility, however, how do I map it? my UI does not seem to support it
<img width="1027" alt="Screen Shot 2021-06-10 at 4 46 55 PM" src="https://user-images.githubusercontent.com/7624943/121610779-9e0c7680-ca0b-11eb-9d2f-2b9f872099d5.png">

**Reply by philleepflorence** [reply]
It looks like you want a _User Interface_ within the [_Directus App_](https://docs.directus.io/concepts/application/) to match the [_Directus API_](https://docs.directus.io/concepts/databases/#dynamic-api) Utils Endpoints?

Consider using [Postman API Client](https://www.postman.com) to _Send_ or _Validate_ API Requests and _View_ or _Debug_ Responses: for any core or custom endpoint.

**Reply by rijkvanzanten** [reply]
Nah, they're saying that there's no Import method in the app to match the export functionality

**Reply by philleepflorence** [reply]
Ah okay, got you...does the recommendation work for interacting with the API Endpoints?

Tried to "translate" to [Directus Concepts](https://docs.directus.io/getting-started/introduction/#modular-extensible).

**Reply by rijkvanzanten** [reply]
> does the recommendation work for interacting with the API Endpoints?

You mean to use Postman to debug / try out API endpoints? If that's the case, then yes! Ideal way to debug / try out APIs 👍🏻

**Reply by arpijoy** [reply]
maybe as an API endpoint is good for engineers, but not good for product managers who are building contents sometimes

**Reply by philleepflorence** [reply]
@arpijoy ... I can understand your use case...

> [Data API Integration](https://rapidapi.com/marketplace) is a huge part of my world, maybe not so much for content contributors or blog creators.

A utility of Postman that has worked for developer, data entry, and content contributor use cases is the ability to share and sync _core_, _custom_, and _external_ (Facebook, Zapier, Google, et al) API endpoints..._although an App UI is preferable for interfacing with each of the API endpoints, it is not currently feasible_. Providing a client some solution has always faired better than, "no you cannot do that", so far.

### Related Discussions
[Import to update from file](https://github.com/directus/directus/discussions/5832)
[How can I import a list of users](https://github.com/directus/directus/discussions/5605)
[Add ability to Import data](https://github.com/directus/directus/discussions/4926)

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
